src/main/kotlin/nl/avisi/structurizr/site/generatr/site/model/PageViewModel.kt

```diff
@@ -1,20 +1,28 @@
 package nl.avisi.structurizr.site.generatr.site.model
 
 import nl.avisi.structurizr.site.generatr.site.GeneratorContext
+import nl.avisi.structurizr.site.generatr.site.model.FlexmarkConfig
 
 abstract class PageViewModel(protected val generatorContext: GeneratorContext) {
     val pageTitle: String by lazy {
         if (pageSubTitle.isNotBlank() && generatorContext.workspace.name.isNotBlank())
-            "$pageSubTitle | ${generatorContext.workspace.name}"
+        "$pageSubTitle | ${generatorContext.workspace.name}"
         else if (generatorContext.workspace.name.isNotBlank())
-            generatorContext.workspace.name
+        generatorContext.workspace.name
         else
-            pageSubTitle
+        pageSubTitle
```

Comment:
Please make sure that this code is properly indented. This is easily done by using the auto formatter of your IDE of choice (again: I recommend using IntelliJ Community edition).

Reply:
done